src/injected/styles/injected.scss

```diff
@@ -11,6 +11,17 @@
     filter: grayscale(100%) !important;
 }
 
+.insights-formatted-text-spacing-container {
+    * {
```

Comment:
This is going to miss text that is a direct child of the `<body>` (eg,

```html
<body>
<h1>this works</h1>
this will be missed
</body>
```

I think you could just drop the `*` and make the 3 styles directly apply to the `.insights-formatted-text-spacing-container`. But if the `*` is important for the sake of precedence rule cases I haven't thought of, you could alternately do `&, * { ... }`